rancher/rancher-ui/applications/stacks/adding-balancers/index.md

```diff
@@ -10,20 +10,99 @@ After adding multiple services to your stack, you might have decided that you wa
+
+### Basic Load Balancing
+
+In the most basic use case, we can select as many listening ports and services as we'd like. We've made the assumption that the access is set as `public` for every listening ports. The source port(s) of the load balancer will use the round robin algorithm to forward traffic to any service(s) on the target port(s). 
+
+**Example:**
+
+When we set up 2 listening ports and 3 services, the load balancer will direct traffic on both listening ports across all services.
+
+Source Port| Listening Port
+---|---
+80 | 80
+443 | 
+<br>
+
+| Target Services|
+|----|
+| Service1 |
+| Service2 |
+|Service3 |
+
+<br>
+In our example, any traffic directed to port `80` on the host of the load balancer would alternate between Service1, Service2 and Service3 on the target port `80`. Any traffic directed to port `443` on the host of the load balancer would alternate between Service1, Service2 and Service3 on port `443`. 
+
+With our basic load balancing, it should solve the most basic use cases. There are limitations within the basic load balancing. If you want to load balance different services (i.e. virtual hosting) or listen using host headers/paths/ports, you can use our advanced routing options for more flexibility.
+
+### Advanced Routing Options
+
+To expand the advanced options, click on the **Show advanced routing options**. When using the advanced options, the only required field is **Target Service**, which would mimic the basic load balancing examples. All of the other fields in the advanced section are optional. For each option, we cover some basic examples, but you can use all or some of these options together to define exactly how you want your load balancer to direct traffic to your services.
+
+#### Request Host/Request Path 
+
+The request host can be a specific HTTP host header used for each service. The request path can be a specific path. The request host and request path can be used independently or in conjunction to create a specific request. 
+
+**Example:**
+
+domain1.com -> Service1 <br>
+domain2.com -> Service2
+
+domain3.com -> Service1 <br>
+domain3.com/admin -> Service2
+
+If no ports are defined for the service, then the source port and target port of the listening ports will be used to direct traffic based on the request. You can also specify the source ports and target ports for each request and the traffic would use the specified ports for the request.
+
+#### Target Port
+
+For each service, you can override the target port defined in the listening port section. This would allow you to direct traffic to services on different ports from the same source port.
+
+**Example**
+
+Source Port| Default Target Port
+---|---
+80 | 80
+<br>
+
+|Target Services| Target Ports|
+|---| ---|
+|Service1| None|
+|Service2 | `443`|
```

Comment:
We should use some other port for the example :) before ssl termination comes in.